cit3.0-web/public/static/env.js

```diff
@@ -0,0 +1,5 @@
+(function (window) {
+  window.env = window.env || {};
+  // Environment variables
+  window.env.apiUrl = "http://localhost:8000";
+})(this);
```

Comment:
Do we want to commit this file as is?

Reply:
Yes, its just so that it works when developing locally